apicast/src/linked_list.lua

```diff
@@ -0,0 +1,45 @@
+local setmetatable = setmetatable
+
+local _M = {
+
+}
+
+local noop = function() end
+
+
+local empty_t = setmetatable({}, { __newindex = noop })
+local __index = function(t,k)
+    return t.current[k] or t.next[k]
+end
+
+local ro_mt = {
+    __index = __index,
+    __newindex = noop,
```

Comment:
I think this should raise instead of fail silently. I think it can be confusing for users of this module to run something like `t[k] = '123'` and see that it does not have any effect and does not raise any error.

Reply:
Good point.